ImageLounge/src/DkGui/DkViewPort.cpp

```diff
@@ -1405,19 +1405,24 @@ void DkViewPort::getPixelInfo(const QPoint &pos)
         return;
 
     QPoint xy = mapToImage(pos);
-
-    if (xy.x() == -1 || xy.y() == -1)
+    if (xy.x() < 0)
         return;
 
-    QColor col = getImage().pixel(xy);
+    // TODO: This is rgb now, but we could display native pixel values too, even CMYK in Qt 6.8.0
+    const QImage &img = getImage();
```

Comment:
```suggestion
    const QImage img = getImage();
```
QImage implements `QImage::QImage(QImage&& other)`, so the returned object gets moved. There should be no need to use reference here.
https://doc.qt.io/qt-6/qimage.html#QImage-1